src/decode-response/convert-response-to-returns-type.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+var abi = require("augur-abi");
+var clone = require("clone");
+
+var convertResponseToReturnsType = function (returnsType, response) {
+  var convertedResponse;
+  if (!returnsType) return response;
+  if (response && response !== "0x") {
```

Comment:
Consider inverting this if into a couple guard clauses:
```
if (!response) return response;
if (response === "0x") return response;
if (response.error) return response;
...
```
This will reduce nesting by a level and make the function easier to read (lower cyclomatic complexity).